README.md

```diff
@@ -179,12 +179,25 @@ Shout out to [@aseemk](https://github.com/aseemk) for donating the name.
 
 #### Doesn't Shell.js do this?
 
-No. 
+No.
+ 
+For those who don't know, [Shell.JS](https://github.com/shelljs/shelljs) is an awesome Node package that implements UNIX shell commands programatically in Javascript. Check it out - really. While Shell.JS was tremendously helpful in figuring out how to accomplish Cash, the two do not really conflict.
 
-For those who don't know, [Shell.js](https://github.com/shelljs/shelljs) is an awesome Node app that implements Unix shell commands programatically in Javascript. Check it out - really. While Shell.js was tremendously helpful in figuring out how to accomplish Cash, the two do not really conflict.
+Shell.JS gives the feel of UNIX commands in a code environmenmt, but aims to impliment the commands in a way that makes sense for a javascript library. This means that many commands return JavaScript objects, and some of the rougher and more dangerous edges of bash have been softened a bit.
```

Comment:
I would be lost without vim spell checking enabled by default for my markdown files